modules/rte_mis_core/rte_mis_core.routing.yml

```diff
@@ -24,3 +24,21 @@ rte_mis_core.config.settings:
     _title: 'RTE MIS Core Settings'
   requirements:
     _permission: 'administer site configuration+define entry class'
+
+# Bulk upload multiple location.
+rte_mis_core.form.add_multiple_location:
+  path: '/rte-mis-core/bulk-upload-multiple-location'
```

Comment:
Pls change it to `/admin/location/bulk-upload`

Reply:
Updated